### PR TITLE
Update definition for react-native : AppState

### PR DESCRIPTION
**Pull request by j6lim**
Added AppState, the unified app state manager for iOS and android

### CODE REVIEW

**Comment by j6lim** [comment]
It was my mistake to push two commits consequently before the first one is reviewed and merged. If you don't mind can you review the second one about ViewPagerAndroid?

**Comment by bgrieder** [comment]
:+1: 

No pb

File: react-native/react-native-tests.tsx [hunk 1]
```diff
@@ -21,7 +21,14 @@ For a list of complete Typescript examples: check https://github.com/bgrieder/RN
...
 import * as React from 'react-native'
-const  { StyleSheet, Text, View } = React
+const  {
+    StyleSheet,
+    Text,
+    View,
+    AppState,
+    AppStateIOS,
+    ViewPagerAndroid
+} = React
```
Comment (DanielRosenwasser) [inline react-native/react-native-tests.tsx]: Why don't you just write

```ts
import {
    StyleSheet,
    Text,
    View,
    AppState,
    AppStateIOS,
    ViewPagerAndroid,
} from React;
```

**Comment by j6lim** [comment]
Fixed the test case code as @DanielRosenwasser suggested.

File: react-native/react-native.d.ts [hunk 2]
```diff
@@ -980,6 +980,31 @@ declare namespace  __React {
     }
 
     /**
+     * @see https://facebook.github.io/react-native/docs/viewpagerandroid.html#props
+     */
+
+
```
Comment (DanielRosenwasser) [inline react-native/react-native.d.ts]: Can you remove the excess newlines?

**Pull request merged**